rhel/Dockerfile

```diff
@@ -10,12 +10,17 @@ ENV KONG_VERSION $KONG_VERSION
 ARG KONG_SHA256="490dfceb423ffdfb0621b06774b880f80ec7d51279f54b10e0fa380eec84135b"
 
 LABEL name="Kong" \
-      vendor="Kong" \
-      version="$KONG_VERSION" \
-      release="1" \
-      url="https://konghq.com" \
-      summary="Next-Generation API Platform for Modern Architectures" \
-      description="Next-Generation API Platform for Modern Architectures"
+    vendor="Kong" \
+    version="$KONG_VERSION" \
```

Comment:
Is it normal that only RHEL has `version` label? Shouldn't it be also propagated to `org.opencontainers.image.version`?

Reply:
the redhat container build process is different than every other container build process. Adding the version label to the other containers is a little more effort than I was up to taking on at this time.

The short version: Kong OSS official docker builds cannot use arg's so we would need to update our automation to sed appropriately

Reply:
Fair enough!